autogpt_platform/frontend/src/components/_new/Text/Text.stories.tsx

```diff
@@ -0,0 +1,171 @@
+        Body Text
+      </h2>
+      <Text variant="lead">Lead</Text>
+      <StoryCode code={`<Text variant="lead">Lead</Text>`} />
+      <div className="flex flex-row gap-8">
+        <Text variant="large">Large</Text>
+        <Text variant="large-medium">Large Medium</Text>
+        <Text variant="large-semibold">Large Semibold</Text>
+      </div>
+      <StoryCode
+        code={`<Text variant="large">Large</Text>
+<Text variant="large-medium">Large Medium</Text>
+<Text variant="large-semibold">Large Semibold</Text>`}
+      />
+      <div className="flex flex-row gap-8">
+        <Text variant="body">Body</Text>
+        <Text variant="body-medium">Body Medium</Text>
+      </div>
+      <StoryCode
+        code={`<Text variant="body">Body</Text>
+<Text variant="body-medium">Body Medium</Text>`}
+      />
+      <div className="flex flex-row gap-8">
+        <Text variant="small">Small</Text>
+        <Text variant="small-medium">Small Medium</Text>
+      </div>
+      <StoryCode
+        code={`<Text variant="small">Small</Text>
+<Text variant="small-medium">Small Medium</Text>`}
+      />
+      <Text variant="subtle">Subtle</Text>
+      <StoryCode code={`<Text variant="subtle">Subtle</Text>`} />
+    </div>
+  );
+}
+
+//=============================================================================
+// Headings Only
+//=============================================================================
+export function Headings() {
+  return (
+    <div className="space-y-8">
+      <Text variant="h1">Heading 1</Text>
+      <Text variant="h2">Heading 2</Text>
+      <Text variant="h3">Heading 3</Text>
+      <Text variant="h4">Heading 4</Text>
+    </div>
+  );
+}
+
+//=============================================================================
+// Body Text Only
+//=============================================================================
+export function BodyText() {
+  return (
+    <div className="space-y-8">
+      <Text variant="lead">Lead</Text>
+      <Text variant="large">Large</Text>
+      <Text variant="large-medium">Large Medium</Text>
+      <Text variant="large-semibold">Large Semibold</Text>
+      <Text variant="body">Body</Text>
+      <Text variant="body-medium">Body Medium</Text>
+      <Text variant="small">Small</Text>
+      <Text variant="small-medium">Small Medium</Text>
+      <Text variant="subtle">Subtle</Text>
+    </div>
+  );
+}
+
+//=============================================================================
+// Interactive Playground
+//=============================================================================
+export const Playground: Story = {
+  args: {
+    variant: "body",
+    children:
+      "Edit this text and try different variants via the controls below",
+  },
+  parameters: {
+    controls: { include: ["variant", "as", "children"] },
+  },
+  render: () => (
+    <div className="space-y-8">
+      <Text variant="h2" as="h2">
```

Comment:
This looks like it doesn't use the interactive controls:

Should it be something like this?

`<Text variant={args.variant} as={args.as}>{args.children}</Text>`